City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal updates for 21.03.2022

## The following packages were updated:
```
  - Upgrading composer/composer (2.2.6 => 2.2.9)
  - Upgrading composer/xdebug-handler (2.0.4 => 3.0.3)
  - Upgrading consolidation/log (2.1.0 => 2.1.1)
  - Upgrading consolidation/site-alias (3.1.4 => 3.1.5)
  - Upgrading doctrine/instantiator (1.4.0 => 1.4.1)
  - Upgrading drupal/core (9.3.6 => 9.3.8)
  - Upgrading drupal/core-composer-scaffold (9.3.6 => 9.3.8)
  - Upgrading drupal/core-dev (9.3.6 => 9.3.8)
  - Upgrading drupal/core-recommended (9.3.6 => 9.3.8)
  - Upgrading drupal/editoria11y (1.0.10 => 1.0.12)
  - Upgrading drupal/entity (1.2.0 => 1.3.0)
  - Upgrading drupal/hdbt (2.2.9 => 2.2.15)
  - Upgrading drupal/hdbt_admin (1.4.6 => 1.4.11)
  - Upgrading drupal/helfi_platform_config (2.5.0 => 2.5.13)
  - Upgrading drupal/helfi_tunnistamo (2.0.2 => 2.0.6)
  - Upgrading drupal/matomo (1.15.0 => 1.17.0)
  - Upgrading drupal/oembed_providers (2.0.1 => 2.0.2)
  - Upgrading drupal/paragraphs (1.12.0 => 1.13.0)
  - Upgrading drupal/simple_sitemap (3.11.0 => 4.1.1)
  - Upgrading drupal/siteimprove (1.11.0 => 1.12.0)
  - Upgrading myclabs/deep-copy (1.10.2 => 1.11.0)
  - Upgrading phpunit/php-code-coverage (9.2.12 => 9.2.15)
  - Upgrading slevomat/coding-standard (7.0.18 => 7.0.19)
  - Upgrading symfony/dom-crawler (v4.4.37 => v4.4.39)
  - Upgrading symfony/filesystem (v4.4.37 => v4.4.39)
  - Upgrading symfony/lock (v4.4.37 => v4.4.38)
  - Upgrading symfony/polyfill-php72 (v1.24.0 => v1.25.0)
  - Upgrading symfony/polyfill-php73 (v1.24.0 => v1.25.0)
```

Round two of updates:
```
  - Upgrading drupal/core (9.3.8 => 9.3.9)
  - Upgrading drupal/core-composer-scaffold (9.3.8 => 9.3.9)
  - Upgrading drupal/core-dev (9.3.8 => 9.3.9)
  - Upgrading drupal/core-recommended (9.3.8 => 9.3.9)
  - Upgrading drupal/hdbt (2.2.15 => 2.3.4)
  - Upgrading drupal/hdbt_admin (1.4.11 => 1.5.1)
  - Upgrading drupal/helfi_platform_config (2.5.13 => 2.6.1)
  - Upgrading guzzlehttp/psr7 (1.8.3 => 1.8.5)
```

## How to test:
- Checkout branch
- Run `make composer-install drush-cr drush-cim drush-updb`
- Make sure all modules are up-to-date: https://helsinki-paatokset.docker.so/fi/admin/reports/updates (The Gin admin theme has a alpha -> beta update, but that will probably be handled in the HDBT project).
- Make sure there are no pending configuration changes: `make drush-cex`
- Click around the site and try creating some content to make sure nothing breaks
